### PR TITLE
Add arn of the glue crawler

### LINKED ISSUE #7934: Provide ARN attribute on Glue Crawler
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

I'm trying to create IAM policies to allow processes to start and describe Glue Crawlers. To scope this tightly, I would like to use the ARN of the crawler, rather than just `*`. The `aws_glue_crawler` resource doesn't provide this attribute, it only provides the id (name) of the resource.

### New or Affected Resource(s)

* aws_glue_crawler

### Potential Terraform Configuration

```hcl
${aws_glue_crawler.my_crawler.arn}"
```

### Current Terraform Configuration

Currently, this is how I'm building the ARN:

```hcl
data "aws_region" "current" {}

data "aws_caller_identity" "current" {}

"arn:aws:glue:${data.aws_region.current.name}:${data.aws_caller_identity.current.account_id}:crawler/${aws_glue_crawler.my_crawler.name}"
```

### PR DESCRIPTION
**Pull request by kterada0509**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" comments, they generate extra noise for pull request followers and do not help prioritize the request

Fixes #7934 

Changes proposed in this pull request:

* Add arn attribute for the glue crawler resource.

Output from acceptance testing:

```
$ make testacc TEST=./aws TESTARGS='-run=TestAccAWSGlueCrawler_'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./aws -v -parallel 20 -run=TestAccAWSGlueCrawler_ -timeout 120m
=== RUN   TestAccAWSGlueCrawler_DynamodbTarget
=== PAUSE TestAccAWSGlueCrawler_DynamodbTarget
=== RUN   TestAccAWSGlueCrawler_JdbcTarget
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget
=== RUN   TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== RUN   TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== PAUSE TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== RUN   TestAccAWSGlueCrawler_S3Target
=== PAUSE TestAccAWSGlueCrawler_S3Target
=== RUN   TestAccAWSGlueCrawler_S3Target_Exclusions
=== PAUSE TestAccAWSGlueCrawler_S3Target_Exclusions
=== RUN   TestAccAWSGlueCrawler_S3Target_Multiple
=== PAUSE TestAccAWSGlueCrawler_S3Target_Multiple
=== RUN   TestAccAWSGlueCrawler_recreates
=== PAUSE TestAccAWSGlueCrawler_recreates
=== RUN   TestAccAWSGlueCrawler_Classifiers
=== PAUSE TestAccAWSGlueCrawler_Classifiers
=== RUN   TestAccAWSGlueCrawler_Configuration
=== PAUSE TestAccAWSGlueCrawler_Configuration
=== RUN   TestAccAWSGlueCrawler_Description
=== PAUSE TestAccAWSGlueCrawler_Description
=== RUN   TestAccAWSGlueCrawler_Role_ARN_NoPath
=== PAUSE TestAccAWSGlueCrawler_Role_ARN_NoPath
=== RUN   TestAccAWSGlueCrawler_Role_ARN_Path
=== PAUSE TestAccAWSGlueCrawler_Role_ARN_Path
=== RUN   TestAccAWSGlueCrawler_Role_Name_Path
=== PAUSE TestAccAWSGlueCrawler_Role_Name_Path
=== RUN   TestAccAWSGlueCrawler_Schedule
=== PAUSE TestAccAWSGlueCrawler_Schedule
=== RUN   TestAccAWSGlueCrawler_SchemaChangePolicy
=== PAUSE TestAccAWSGlueCrawler_SchemaChangePolicy
=== RUN   TestAccAWSGlueCrawler_TablePrefix
=== PAUSE TestAccAWSGlueCrawler_TablePrefix
=== RUN   TestAccAWSGlueCrawler_SecurityConfiguration
=== PAUSE TestAccAWSGlueCrawler_SecurityConfiguration
=== CONT  TestAccAWSGlueCrawler_Schedule
=== CONT  TestAccAWSGlueCrawler_SecurityConfiguration
=== CONT  TestAccAWSGlueCrawler_DynamodbTarget
=== CONT  TestAccAWSGlueCrawler_Role_Name_Path
=== CONT  TestAccAWSGlueCrawler_Role_ARN_Path
=== CONT  TestAccAWSGlueCrawler_Role_ARN_NoPath
=== CONT  TestAccAWSGlueCrawler_Description
=== CONT  TestAccAWSGlueCrawler_Configuration
=== CONT  TestAccAWSGlueCrawler_Classifiers
=== CONT  TestAccAWSGlueCrawler_recreates
=== CONT  TestAccAWSGlueCrawler_S3Target_Multiple
=== CONT  TestAccAWSGlueCrawler_S3Target_Exclusions
=== CONT  TestAccAWSGlueCrawler_S3Target
=== CONT  TestAccAWSGlueCrawler_JdbcTarget_Multiple
=== CONT  TestAccAWSGlueCrawler_JdbcTarget_Exclusions
=== CONT  TestAccAWSGlueCrawler_TablePrefix
=== CONT  TestAccAWSGlueCrawler_JdbcTarget
=== CONT  TestAccAWSGlueCrawler_SchemaChangePolicy
--- PASS: TestAccAWSGlueCrawler_Role_ARN_NoPath (54.94s)
--- PASS: TestAccAWSGlueCrawler_recreates (57.04s)
--- PASS: TestAccAWSGlueCrawler_Role_Name_Path (58.20s)
--- PASS: TestAccAWSGlueCrawler_Role_ARN_Path (66.06s)
--- PASS: TestAccAWSGlueCrawler_Configuration (76.62s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Exclusions (77.19s)
--- PASS: TestAccAWSGlueCrawler_Description (81.93s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Exclusions (82.54s)
--- PASS: TestAccAWSGlueCrawler_S3Target (85.16s)
--- PASS: TestAccAWSGlueCrawler_SecurityConfiguration (85.33s)
--- PASS: TestAccAWSGlueCrawler_SchemaChangePolicy (91.27s)
--- PASS: TestAccAWSGlueCrawler_TablePrefix (93.95s)
--- PASS: TestAccAWSGlueCrawler_Schedule (94.12s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget (94.81s)
--- PASS: TestAccAWSGlueCrawler_JdbcTarget_Multiple (104.17s)
--- PASS: TestAccAWSGlueCrawler_DynamodbTarget (108.94s)
--- PASS: TestAccAWSGlueCrawler_Classifiers (113.93s)
--- PASS: TestAccAWSGlueCrawler_S3Target_Multiple (116.88s)
PASS
ok  	github.com/terraform-providers/terraform-provider-aws/aws	116.966s
```

### CODE REVIEW

File: aws/resource_aws_glue_crawler.go [hunk 1]
```diff
@@ -459,6 +463,14 @@ func resourceAwsGlueCrawlerRead(d *schema.ResourceData, meta interface{}) error
 		return nil
 	}
 
+	arn := arn.ARN{
```
Comment (bflad) [inline aws/resource_aws_glue_crawler.go]: I would be careful to not shadow the import name by calling this `crawlerARN :=` or something. 👍

Comment (kterada0509) [reply aws/resource_aws_glue_crawler.go]: Thanks, fixed.

**Comment by kterada0509** [comment]
> This is looking good, @kterada0509, can you please also add this resource to the list of `skip_requesting_account_id` resources in `website/docs/index.html.markdown`? Much appreciated!

@bflad Fixed it.

**Comment by bflad** [comment]
This has been released in [version 2.2.0 of the Terraform AWS provider](https://github.com/terraform-providers/terraform-provider-aws/blob/master/CHANGELOG.md#220-march-15-2019). Please see the [Terraform documentation on provider versioning](https://www.terraform.io/docs/configuration/providers.html#provider-versions) or reach out if you need any assistance upgrading.

**Pull request merged**